corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -184,6 +185,7 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
                     "isShortDetail": params.isShortDetail,
                     "isRefreshCaseSearch": params.isRefreshCaseSearch,
                     "requestInitiatedByTag": params.requestInitiatedByTag,
+                    "screen_size": screenSize,
```

Comment:
`windowWidth` is a good idea. When I started on this it wasn't clear exactly what the value was going to be, but since it turned out to be the actual width, as opposed to a boolean or something else, `windowWidth` makes sense.

As far as the snake_case goes, I was following what seemed like a standard pattern in FP's `SessionNavigationBean` which extends `InstallRequestBean` which extends `AuthenticatedRequestBean`. That turns out to not be consistent when you look at `restoreAsCaseId` in  `AuthenticatedRequestBean` so I'm not sure it matters too much. camelCase also doesn't break anything in FP so it seems fine to use.

Reply:
I was just looking at the line above as reference.